db/versions/0079.yml

```diff
@@ -0,0 +1,41 @@
+version: 79
+description: Speed up hooks fetching last fires by date
+migrationScript: |-
+  begin
+    create index hooks_last_fires_time on hooks_last_fires (hook_group_id,hook_id,task_create_time);
```

Comment:
This was tested on the production dataset locally, which has over 2.000.000 records in that table.
Index creation is fast, and overhead is few hundred MB in storage